### ISSUE TITLE
PHP Fatal error:  Uncaught ValueError: mb_convert_encoding(): Argument #3 ($from_encoding) contains invalid encoding "WINDOWS-1253" in /var/www/localhost/htdocs/roundcube/program/lib/Roundcube/rcube_charset.php:297

### ISSUE THREAD

**Issue by candrews** [roundcube/roundcubemail#7850]
I'm getting this fatal error when viewing the message list in Roudcube using the current master 5975742bcb37edd6367ff4a929dcda6017679de5

```
Jan 25 15:58:36 irrational roundcube[221895]: PHP Fatal error:  Uncaught ValueError: mb_convert_encoding(): Argument #3 ($from_encoding) contains invalid encoding "WINDOWS-1253" in /var/www/localhost/htdocs/roundcube/program/lib/Roundcube/rcube_charset.php:297
Jan 25 15:58:36 irrational roundcube[221895]: Stack trace:
Jan 25 15:58:36 irrational roundcube[221895]: #0 /var/www/localhost/htdocs/roundcube/program/lib/Roundcube/rcube_charset.php(297): mb_convert_encoding()
Jan 25 15:58:36 irrational roundcube[221895]: #1 /var/www/localhost/htdocs/roundcube/program/lib/Roundcube/rcube_mime.php(251): rcube_charset::convert()
Jan 25 15:58:36 irrational roundcube[221895]: #2 /var/www/localhost/htdocs/roundcube/program/lib/Roundcube/rcube_mime.php(154): rcube_mime::decode_mime_string()
Jan 25 15:58:36 irrational roundcube[221895]: #3 /var/www/localhost/htdocs/roundcube/program/actions/mail/index.php(527): rcube_mime::decode_header()
Jan 25 15:58:36 irrational roundcube[221895]: #4 /var/www/localhost/htdocs/roundcube/program/actions/mail/list.php(122): rcmail_action_mail_index::js_message_list()
Jan 25 15:58:36 irrational roundcube[221895]: #5 /var/www/localhost/htdocs/roundcube/program/include/rcmail.php(275): rcmail_action_mail_list->run()
Jan 25 15:58:36 irrational roundcube[221895]: #6 /var/www/localhost/htdocs/roundcube/index.php(279): rcmail->action_handler()
Jan 25 15:58:36 irrational roundcube[221895]: #7 {main}
Jan 25 15:58:36 irrational roundcube[221895]:   thrown in /var/www/localhost/htdocs/roundcube/program/lib/Roundcube/rcube_charset.php on line 297
```

```
$ php --version
PHP 8.0.1 (cli) (built: Jan  8 2021 09:15:07) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.1, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.1, Copyright (c), by Zend Technologies
```

**Comment by alecpl** [comment]
I'm so disappointed about PHP8.

**Comment by alecpl** [comment]
Fixed via d258f845ab2e.
